features/step_definitions/ontology_state_steps.rb

```diff
@@ -1,17 +1,25 @@
 Given(/^There is an ontology$/) do
   @ontology = FactoryGirl.create :ontology
-  @user = FactoryGirl.create :admin
-  login_as @user, scope: :user
+end
+
+Given(/^there is an ontology with a pending version$/) do
+  @ontology_version = FactoryGirl.create :ontology_version, state: 'pending'
+  @ontology = @ontology_version.ontology
 end
 
 Given(/^I visit the detail page of the ontology$/) do
-  pending("Tim knows what is to do and said i should make this comment")
+  visit locid_for(@ontology, :ontology_versions)
 end
 
-When(/^now the state will be updated$/) do
-  pending
+When(/^we change the state of the ontology to: (\w+)$/) do |state|
+  @state = state
+  @ontology_version.update_state!(state)
 end
 
 Then(/^the page should change the state on its own$/) do
-  pending
+  el_css = 'small.ontology-version-state'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.